pycparser/c_parser.py

```diff
@@ -480,6 +480,13 @@ def _select_struct_union_class(self, token):
         else:
             return c_ast.Union
 
+    def _to_list(self, p):
+        """ Normalize the input to a list: if given a list, return as-is, 
```

Comment:
I'd rather have this refactoring in a separate PR that can come earlier. It obscures the actual changes in this PR